regparser/grammar/unified.py

```diff
@@ -72,6 +74,22 @@ def appendix_section(match):
                     | atomic.upper_p)
        ).setParseAction(appendix_section).setResultsName("appendix_section"))
 
+# "the" appendix implies there's only one, so it better be appendix A
+section_of_appendix_to_this_part = (
+    atomic.section_marker
+    + atomic.upper_roman_a.setResultsName("appendix_section")
```

Comment:
Note that you always want to `copy()` the grammar bits. Here, you are renaming the result in `upper_roman_a` across the app.